ext/mkrf_conf.rb

```diff
@@ -37,7 +37,7 @@
   os == 'ios_xr' || deps << Gem::Dependency.new('net_http_unix',
                                                 '~> 0.2', '>= 0.2.1')
   # NX-OS doesn't need gRPC
-  os == 'nexus' || deps << Gem::Dependency.new('grpc', '~> 0.12')
+  os == 'nexus' || deps << Gem::Dependency.new('grpc', '~> 1.14.1')
```

Comment:
Are you testing XR in your environment?

Reply:
We are not testing / using this code w/ XR at all (to my knowledge)

Reply:
Ok.. just wondering why the version bump?  We have considered removing support for this.

Reply:
When using Puppet Device it installs both gems.  This is basically a temporary fix to unblock agentless.  I agree we should probably remove the gRPC / XR stuff in the future.